src_labels/index.js

```diff
@@ -33,7 +33,7 @@ const files = getLabelFilesFrom(rootDir);
 const output = {};
 for(let i=0; i < files.length; i++) {
 	const f = files[i];
-	const keys = f.replace(/\\+/g, "\\").replace(rootDir, "").replace(/\.json$/gi, "").split("\\");
+	const keys = f.replace(new RegExp(`${path.sep}+`.replace('\\', '\\\\'), 'g'), path.sep).replace(rootDir, "").replace(/\.json$/gi, "").split(path.sep);
```

Comment:
I just checkd and the first replace was actually not necessary
When logging the files i saw double "\\" everywhere but that was just du to logging.
Only doing this works on windows:
```javascript
f.replace(rootDir, "").replace(/\.json$/gi, "").split(path.sep);
```
If you can make sure it works on unix system as well that's perfect!
But i guess it will :)